drivers/hmis/app/models/hmis/ce/match/README_FOR_CE_MATCH_ENGINE.md

```diff
@@ -28,3 +28,16 @@ The classes within this module are organized into several subdirectories to grou
     * Creates or updates candidate records for the pool for newly matched clients or where priority score has changed.
     * Deletes candidate records for clients that no-longer match the pool's expression
 5.  **Event Logging**: The `CandidateEventWriter` records the outcome of the evaluation (`add`, `update`, `remove`) in the `ce_match_candidate_events` table, creating an audit trail. An event is only logged when there is a meaningful change in the candidate's status: when they are first added, when their priority score changes, or when they are removed from the pool.
+
+## Concepts and Contracts (Quick Reference)
+
+- **Candidate Pool Key**: `[priority_expression, requirement_expression]` (Strings). Default/no‑specific‑rules returns `nil` (no pool is created; associations remain `NULL`).
+- **Rule Precedence**: Priority scheme selection is by owner precedence `UnitGroup > Project > Organization`; tie-breaker by rule `id`. Eligibility requirements from all applicable owners are combined with logical AND in deterministic order.
```

Comment:
Not sure this is the right place, but somewhere we should note that per-unit rules should be implemented by putting the units in individual unit groups.  This was our work-around that allows us to streamline the places candidate pools are attached.